src/main/java/com/authlete/common/api/AuthleteApi.java

```diff
@@ -416,7 +542,7 @@ public interface AuthleteApi
      * </p>
      *
      * <pre>
-     * int totalCount = api.{@link #getTokenList(String, String, int, int)
+     * int totalCount = api.{@link #getTokenList(String, String, int, int, TokenStatus)
      * getTokenList(clientIdentifier, subject, 0, 0)}.{@link TokenListResponse#getTotalCount()
```

Comment:
`#getTokenList(String, String, int, int, TokenStatus)` does not match `getTokenList(clientIdentifier, subject, 0, 0)`.